src/Apps/Legion/Components/ConsumeShell/ConsumeShell.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { getDebugLogger } from '../../../../Models/Services/Utils';
+import { IConsumeShellProps } from './ConsumeShell.types';
+import { gridBorderVar, useClassNames } from './ConsumeShell.styles';
+import { useExtendedTheme } from '../../../../Models/Hooks/useExtendedTheme';
+import { GlobeRegular } from '@fluentui/react-icons';
+import { useAppDataContext } from '../../Contexts/AppDataContext/AppDataContext';
+
+const debugLogging = false;
+const logDebugConsole = getDebugLogger('ConsumeShell', debugLogging);
+
+const ConsumeShell: React.FC<IConsumeShellProps> = (_props) => {
+    // contexts
+
+    // state
+
+    // hooks
+    const theme = useExtendedTheme();
+    const { appDataState } = useAppDataContext();
+
+    // callbacks
+
+    // side effects
+
+    // styles
+    const classNames = useClassNames();
+
+    logDebugConsole('debug', 'Render');
+
+    return (
+        <div className={classNames.root}>
+            <div
+                className={classNames.header}
+                style={{ [`${gridBorderVar}`]: theme.palette.glassyBorder }}
+            >
+                <GlobeRegular fontSize={'18px'} />
+                <h3 className={classNames.headerText}>
+                    {appDataState?.targetDatabase?.databaseName}
+                </h3>
+            </div>
+            <div
+                className={classNames.leftNav}
+                style={{ [`${gridBorderVar}`]: theme.palette.glassyBorder }}
```

Comment:
do you need it in the special ticks? or can it just be the variable directly?

Reply:
I tried using the variable directly, but typescript had issues with understanding it was a string. I can try to type it and see if that works.